.github/workflows/build_dynamo_samples_net8.0.yml

```diff
@@ -0,0 +1,73 @@
+# Build DynamoAll.sln with .NET 8.0
+name: Build DynamoSamples.sln net8.0
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+ build:
+  runs-on: windows-latest
+  steps:
+    - name: Checkout DynamoSamples Repo
+      uses: actions/checkout@v4
+      with:
+        path: DynamoSamples
+        repository: DynamoDS/DynamoSamples
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '8.0.x'
+    - name: Disable problem matcher
+      run: echo "::remove-matcher owner=csc::"
+    - name: Install dependencies for SampleIntegration
+      run: |
+        dotnet restore $Env:GITHUB_WORKSPACE\DynamoSamples\src\SampleIntegration\SampleIntegration.sln /p:Configuration=Release --runtime=win-x64 -p:DotNet=net8.0
```

Comment:
is `-p:DotNet=...` the same as `/p:DotNet=...`?

Reply:
Yes..but dotnet restore has a different format for properties than msbuild

Reply:
sorry, why are you using both formats here?

Reply:
?

Reply:
We do not need the `-p:DotNet=net8.0` here. Removed
Looks like dotnet restore requires /p: for supplying properties
--runtime=win-x64 is separate from properties.